### PR TITLE
[RA-1 Ch03] Section 3.4.1: Virtualisation

### PR DESCRIPTION
**Pull request by msidana**
Creating a draft PR to seek some inputs on the content requirements.
@pgoyal01 @iangardner22 Kindly let me know about expected requirements for this section. I have started with some initial content.

### CODE REVIEW

File: doc/ref_arch/openstack/chapters/chapter03.md [hunk 1]
```diff
@@ -210,8 +210,9 @@ Content to be developed
...
 
 <a name="3.4.1"></a>
-### 3.4.1. Virtualisation (KVM/QEMU)
-Content to be developed
+### 3.4.1. Virtualisation
+Virtualisation is a technology that enables a guest Operating System (OS) to be abstracted from the underlying hardware and software. This allow to run multiple VIrtual Machines(VMs) on the same hardware. Each such VMs have their own OS and are isolated from each other - application running on one VM does not have the access to resources of another VM. Such virtualisation is supported by various hypervisors available as open-source (KVM, Xen etc.) as well as commercial (Hyper-V, Citrix XenServer etc.). Selecting a hypervisor depends on the workload needs and the features provided by various hypervisors as illustrated in Hypervisor [Feature Support Matrix](https://docs.openstack.org/nova/latest/user/support-matrix.html). Openstack (Nova) allows to use various hypervisors within a single installation by means of scheduler filters like ComputeFilter, ImagePropertiesFilter etc.
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter03.md]: @msidana some minor points.
The first sentence is a little confusing. Should this be simply, "Virtualisation technology abstracts the underlying physical hardware allowing multiple virtual servers (machines) to run on the physical server."

"Each such VMs have their own OS and are isolated from each other - application running on one VM does not have the access to resources of another VM. " Wouldn't this depend upon the type of hypervisor (full, para, etc.)

Typos:
VIrtual
OpenStack 
and in "Openstack (Nova) allows to use various ..." should this be "OpenStack (Nova) allows the use of various ..."

Comment (msidana) [reply doc/ref_arch/openstack/chapters/chapter03.md]: ``` "Virtualisation technology abstracts the underlying physical hardware allowing multiple virtual servers (machines) to run on the physical server."``` -> Virtualisation may not necessarily abstract **underlying physical hardware only**. For example, KVM+QEMU enables us to run VMs on VMs. Thus I mentioned ```... that enables a guest Operating System (OS) to be abstracted from the underlying hardware and software``` to keep it generic without going into specifics like type1, type2 hypervisor(s), para/full virtualisation etc.

Corrected typos.

**Comment by pgoyal01** [comment]
Fixes #636

**Comment by msidana** [comment]
@rabiabdel @iangardner22 The comments are resolved. Pls review/approve.

**Comment by msidana** [comment]
> Minor typos:
> 
> "This allow" -change_to- "This allows"
> "Vurtual" - change_to- "Virtual"
> "application running on one VM" -change_to- "i.e. application running on one VM"

Fixed. Thanks !

**Comment by rabiabdel** [comment]
i see @iangardner22 requestrs has been addressed already. approving and merging.

**Pull request merged**